src/utils/css/utils.ts

```diff
@@ -9,21 +9,54 @@ export const clearfix = css`
   }
 `;
 
+/**
+ * Return CSS compatible string
+ * @param cssValue number, string or {value: number, unit: string | null}
+ */
+export const cssValueToString = (
```

Comment:
Another option could be to just export all tokens as css ready-to-use from tokens repo. This way we could always directly use the tokens without having an extra layer of complexity. This should be handled separately though, in case feasible.

Reply:
As discussed, this is a topic for another issue once we have figured out what is actually needed.